source/_docs/automation/trigger.markdown

```diff
@@ -7,20 +7,31 @@ Triggers are what starts the processing of an automation rule. When _any_ of the
 
 An automation can be triggered by an event, with a certain entity state, at a given time, and more. These can be specified directly or more flexible via templates. It is also possible to specify multiple triggers for one automation.
 
+- [Trigger id](#trigger-id)
+- [Trigger variables](#trigger-variables)
 - [Event trigger](#event-trigger)
 - [Home Assistant trigger](#home-assistant-trigger)
 - [MQTT trigger](#mqtt-trigger)
 - [Numeric state trigger](#numeric-state-trigger)
 - [State trigger](#state-trigger)
+  - [Holding a state](#holding-a-state)
 - [Sun trigger](#sun-trigger)
+  - [Sunset / Sunrise trigger](#sunset--sunrise-trigger)
+  - [Sun elevation trigger](#sun-elevation-trigger)
 - [Tag trigger](#tag-trigger)
 - [Template trigger](#template-trigger)
 - [Time trigger](#time-trigger)
+  - [Time String](#time-string)
+  - [Input Datetime](#input-datetime)
+  - [Sensors of datetime device class](#sensors-of-datetime-device-class)
+  - [Multiple Times](#multiple-times)
 - [Time pattern trigger](#time-pattern-trigger)
 - [Webhook trigger](#webhook-trigger)
 - [Zone trigger](#zone-trigger)
 - [Geolocation trigger](#geolocation-trigger)
 - [Device triggers](#device-triggers)
+- [Multiple triggers](#multiple-triggers)
+- [Multiple Entity IDs for the same Trigger](#multiple-entity-ids-for-the-same-trigger)
```

Comment:
```suggestion
```

Reply:
restored accidental tox changes.

Reply:
I quit using the editor in the Docker container, every time I saved a change it changed TOC lines.
Now I just use the editor to VIEW the document.  (checking to see if any rules are violated)